File: Task10/src/app/models/plane.ts
export class Plane{
    constructor(
        public id?: number,
        public Name?: string,
        public ReleaseDate?: Date,
        public LifeTime?: Date,
        public LastHeavyMaintence?: Date,
        public FlightHours?: number,
        public PlaneTypeId?: number
    ){}
}

File: Task11/AirportUI/Models/Entities/Plane.cs
using System;

namespace AirportUI.Models.Entities
{
    public class Plane
    {
        public long Id { get; set; }
        public string Name { get; set; }
        public long TypeId { get; set; }
        public DateTime ReleaseDate { get; set; }
        public DateTime LastHeavyMaintenance { get; set; }
        public int FlightHours { get; set; }
        public TimeSpan Lifetime { get; set; }

        public Plane Clone() => (Plane)MemberwiseClone();
    }
}
